pipdeptree.py

```diff
@@ -314,7 +317,7 @@ def get_children(self, node_key):
         node = self.get_node_as_parent(node_key)
         return self._obj[node] if node else []
 
-    def filter(self, include, exclude):
+    def filter(self, include, exclude, include_re=None, exclude_re=None):
```

Comment:
I can only see `exclude_re` used. Where is include_re being used?

Reply:
Not implemented ... TODO ...